app/src/main/java/co/condorlabs/customcomponents/Exceptions.kt

```diff
@@ -30,3 +30,5 @@ class SkeletonTypeNotSupportedException : RuntimeException("You should provide a
 class StyleNotFoundForType(type: Int) : RuntimeException("We could not found a style for $type")
 
 class FileSelectorViewOptionsNotFound : RuntimeException("You should provide options to open camera or gallery")
+
+class PhoneDigitPositionException : IndexOutOfBoundsException()
```

Comment:
What's the difference between this one and IndexOutOfBoundsException?

Reply:
Is just con _contextualize_ the dev about it